content/future.html

```diff
@@ -2,104 +2,299 @@ <h1 id="future"> Future </h1>
 </p>
 <div class="half second">
-  <span class="graph-title">Factors affecting post-graduation plans</span>
-  <div id="motivation"></div>
+  <span class="graph-title">New Grad Total First-Year Compensation</span>
+  <div id="ft-total"></div>
 </div>
 
 <div class="vbx2"></div>
+<p class="half">
+  <span class="big">
+    The hourly wage / salary will have a median and an average of $81 and
+    $105 CAD respectively.
+  </span>
+  We can again estimate this to be $76 and $99 CAD with today's exchange rate.
+</p>
+<div class="half second">
+  <span class="graph-title">New Grad Hourly Wage</span>
+  <div id="ft-hourly"></div>
+</div>
 
-<h3> Debt </h3>
+<div class="vbx2"></div>
 <p class="half">
   <span class="big">
-    65% of students are graduating debt-free.
+    There's a huge discrepency in profit sharing for new grads,
+    with the median and average being $32,500 and $95,392 CAD.
   </span>
-  30% of students have less than $30,000 in debt. Among students who are
-  in debt, the median debt is $22,500.
+  There are extreme outliers that skewed the results greatly.
   <br /><br />
-  It's important to note that there were students from each family income bracket
-  who graduated debt-free.
-  All students with family income higher than $200,000 graduated debt-free.
+  With the new rate, the results are $30,505 and $89,742 CAD.
 </p>
 <div class="half second">
-  <span class="graph-title">Debt per family income bracket</span>
-  <div id="debt"></div>
+  <span class="graph-title">New Grad Profit Sharing in Compensation</span>
+  <div id="ft-stock"></div>
 </div>
 
 <div class="vbx2"></div>
+<p class="half">
+  <span class="big">
+    Sign-on bonuses were common for respondents, with the median and
+    average being $40,000 and $62,979 CAD.
+  </span>
+  The mode response was $0 for a signing bonus with 15 responses out of
+  the total 47.
+  <br /><br />
+  Adjusting for the new exchange rate, we find the median and average to be $37,630 and $59,248 CAD.
+</p>
+<div class="half second">
+  <span class="graph-title">New Grad Sign-On Compensation</span>
+  <div id="ft-signing"></div>
+</div>
 
-<h3> Advice </h3>
-<p class="full">
+
+<div class="vbx2"></div>
+
+<h3> Motivations </h3>
+<p class="half">
   <span class="big">
-    "Figure out which advice applies best to you and ignore the rest."
+    An enjoyable workplace and good compensation were the biggest factors in
+    accepting full-time roles. 
   </span>
+  It is great to see the class value a positive environment above compensation.
+  <br /><br />
+  In constrast to most other industries, software recruitment has remained
+  relatively unaffected in the face of a global pandemic. Only 1 respondant factored
+  job security in their decision-making. 
+</p>
+<div class="half second">
+  <span class="graph-title">Factors affecting post-graduation plans</span>
+  <div id="motivation"></div>
+</div>
+
+<div class="vbx2"></div>
+
+<h3> Leaving Thoughts </h3>
```

Comment:
```suggestion
<h3> Closing Thoughts </h3>
```

or closing remarks?

i'm assuming we'll leave the future/post-grad section as the last section?

Reply:
yup it'll be the last section!